lib/msal-node/src/utils/RetryTracker.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Logger } from "@azure/msal-common";
+
+export class RetryTracker {
+    private logger: Logger;
+    private retryLimit: number;
+    private retryWaitTimeMilli: number;
+
+    private _numRetried: number;
+    public resetNumRetried(): void {
```

Comment:
Is this needed?

Reply:
I use it in BaseManagedIdentitySource on line 173 when the retry limit has been reached (if applicable).